.github/workflows/create-release.yml

```diff
@@ -1,279 +1,514 @@
+          previous_run_nodejs_16_sha256_image_hash_code=$(cat ${{ github.workspace }}/${{ env.UBI_RUN_NODEJS_16_SHA256_FILENAME }})
+          ubi_image_previous_hash_codes+=("node-16-run" $previous_run_nodejs_16_sha256_image_hash_code)
+
+          if [ "$previous_run_nodejs_16_sha256_image_hash_code" != "$current_run_nodejs_16_sha256_image_hash_code" ]; then
+            ubi_images_need_update+=("node-16-run")
+          fi
+
+          # Run Image Node.js 18
+          docker pull registry.access.redhat.com/ubi8/nodejs-18-minimal
+          current_run_nodejs_18_sha256_image_hash_code=$(docker image inspect registry.access.redhat.com/ubi8/nodejs-18-minimal --format '{{json .RepoDigests}}' | jq -r '.[0]')
+          previous_run_nodejs_18_sha256_image_hash_code=$(cat ${{ github.workspace }}/${{ env.UBI_RUN_NODEJS_18_SHA256_FILENAME }})
+          ubi_image_previous_hash_codes+=("node-18-run" $previous_run_nodejs_18_sha256_image_hash_code)
+
+          if [ "$previous_run_nodejs_18_sha256_image_hash_code" != "$current_run_nodejs_18_sha256_image_hash_code" ]; then
+            ubi_images_need_update+=("node-18-run")
+          fi
+
+          json_ubi_images_need_update=$(jq --compact-output --null-input '$ARGS.positional' --args -- "${ubi_images_need_update[@]}")
+          printf "ubi_images_need_update=%s\n" "${json_ubi_images_need_update}" >> "$GITHUB_OUTPUT"
+
+          json_ubi_image_previous_hash_codes=$(jq --compact-output --null-input '$ARGS.positional' --args -- "${ubi_image_previous_hash_codes[@]}")
+          printf "ubi_image_previous_hash_codes=%s\n" "${json_ubi_image_previous_hash_codes}" >> "$GITHUB_OUTPUT"
 
   create_stack:
     name: Create Stack
+    needs: poll_ubi_images
+    if: ${{ ! ( needs.poll_ubi_images.outputs.ubi_images_need_update == '[]' && github.event_name == 'schedule' ) }}
+
     runs-on: ubuntu-22.04
     steps:
-    - name: Checkout
-      uses: actions/checkout@v3
-
-    - name: Create stack
-      id: create-stack
-      uses: paketo-buildpacks/github-config/actions/stack/create-stack@main
-
-    - name: Upload build image
-      uses: actions/upload-artifact@v3
-      with:
-        name: current-build-image
-        path: build/build.oci
-
-    - name: Upload base run image
-      uses: actions/upload-artifact@v3
-      with:
-        name: current-run-image
-        path: build/run.oci
-
-    - name: Upload nodejs-16 run image
-      uses: actions/upload-artifact@v3
-      with:
-        name: current-nodejs-16-run-image
-        path: build-nodejs-16/run.oci
-
-    - name: Upload nodejs-18 run image
-      uses: actions/upload-artifact@v3
-      with:
-        name: current-nodejs-18-run-image
-        path: build-nodejs-18/run.oci
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Create stack
+        id: create-stack
+        run: |
```

Comment:
I think we should still be able to use  uses: paketo-buildpacks/github-config/actions/stack/create-stack@main versus calling the ./scripts/create.sh directly. When possible we want to use the comment actions components.

Reply:
We removed the action from the github-config repo because we weren't using it anymore.

Reply:
ok got it